sub-packages/bionemo-geneformer/tests/bionemo/geneformer/test_model.py

```diff
@@ -260,6 +260,7 @@ def __getitem__(self, idx):
         return {"text": self.input_ids[idx], "attention_mask": self.mask[idx]}
 
 
+@pytest.mark.xfail(reason="Known issue on H100 GPUs")
```

Comment:
```suggestion
@pytest.mark.xfail("H100" in torch.cuda.get_device_name(0), reason="Known issue on H100 GPUs")
```
Can we xfail this only if we're on an H100? (confirm this works)

Reply:
Agreed. Also you can do something like this instead:

```
from transformer_engine.pytorch.utils import get_cudnn_version
from transformer_engine.pytorch.utils import get_device_compute_capability

@pytest.mark.xfail(get_cudnn_version() < (9, 5, 0) and get_device_compute_capability() >= (9, 0), reason="Known issue on hopper GPUs with older cuDNN versions pre 9.5.")
```

Reply:
When we're on hopper we see really good precision with cudnn >= 9.5.0 at least, so I'm guessing it would work in that case.

Reply:
Done, checking for Hx00 GPU now